lib/view_component/base.rb

```diff
@@ -91,14 +91,21 @@ def render_in(view_context, &block)
       before_render
 
       if render?
-        render_template_for(@variant)
+        render_template_for(@variant) + _output_postamble
```

Comment:
It looks like with this change means `call` must return a string (or something that works with `+` at least)?

In our usage, we've overridden `call` a couple places (using the [inline rendering](https://viewcomponent.org/guide/templates.html#inline)) and happened to return some other things like `nil` (or the contents of a slot, an instance of ViewComponent::SlotV2). I guess something later in the pipeline string-ified them but now breaks with `undefined method `+' for nil:NilClass`.

We might be in the wrong, but would it be useful to clarify in the documentation that `call` must return a string at least?